src/logging/timer.h

```diff
@@ -93,6 +95,8 @@ class Timer
 } // namespace BCLog
 
 
+#define LOG_TIME_MICROS_WITH_CATEGORY(end_msg, log_category) \
```

Comment:
please update this one:

```diff
-#include <sync.h>
+#include <util/macros.h>
```

Otherwise PR causes:
```
A new circular dependency in the form of "logging/timer -> sync -> logging/timer" appears to have been introduced.
```